tests/kernel/pipe/pipe_api/prj.conf

```diff
@@ -3,3 +3,4 @@ CONFIG_IRQ_OFFLOAD=y
 CONFIG_TEST_USERSPACE=y
 CONFIG_DYNAMIC_OBJECTS=y
 CONFIG_MP_NUM_CPUS=1
+CONFIG_USERSPACE=y
```

Comment:
Don't force userspace. This test suite needs to also work without userspace.

Reply:
I add one more test in yaml which enables userspace instead.